src/entities/Event/routes/updateEvent.ts

```diff
@@ -353,6 +336,43 @@ export async function updateEvent(req: WithAuthProfile<WithAuth>) {
 
   await EventModel.update(updatedAttributes, { id: event.id })
 
+  // Notify community members when:
+  // 1. Event is approved AND has a community
+  // 2. Event is being approved for the first time OR community is changing to a new one
+  // 3. Community is not being explicitly removed (req.body.community_id !== null)
+  const shouldNotify =
+    updatedAttributes.approved && // Event is approved
+    updatedAttributes.community_id && // Event has a community
+    req.body.community_id !== null && // Community is not being explicitly removed
+    (!event.approved || // Event is being approved for the first time
+      (event.approved && updatedAttributes.community_id !== event.community_id)) // OR event was already approved but community changed
+
+  if (shouldNotify) {
+    try {
+      const community = await Communities.get().getCommunity(
+        updatedAttributes.community_id!
+      )
+
+      if (community) {
+        console.log(
+          "Sending community notification for community:",
+          community.name
+        )
+        // do not fail the event update if the notification fails
+        await notifyCommunityMembers(updatedEvent, community).catch((error) => {
+          console.error("Failed to send community notification:", error)
+        })
```

Comment:
In theory, this catch should not be needed anymore, right?
```suggestion
        await notifyCommunityMembers(updatedEvent, community)
```